runshell.py

```diff
@@ -8,12 +8,12 @@
 def runshell():
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "tests.settings")
     execute_from_command_line(
-        sys.argv[:1] +
-        ['migrate', '--noinput', '-v', '0'] +
-        (['--run-syncdb'] if django.VERSION >= (1, 9) else []))
+        sys.argv[:1] + ["migrate", "--noinput", "-v", "0"] + (["--run-syncdb"] if django.VERSION >= (1, 9) else [])
+    )
```

Comment:
This can be simplified, as Django 1.9 is long out of date.

Reply:
Sure, will raise a separate PR. It's not really a formatting issue.